swiftglue/Makefile

```diff
@@ -37,17 +37,27 @@ $(3)_SDK:=$$(shell xcrun --sdk $(2) --show-sdk-path)
 
 ## debug
 $(3)_DEBUG_OUTPUTDIR:=$(BINDIR)/Debug/$(1)/$$($(3)_ARCH)
-$$($(3)_DEBUG_OUTPUTDIR)/$(MODULENAME): Makefile $(wildcard *.swift)
+$(3)_DEBUG_COUTPUTDIR:=$(BINDIR)/Debug/$(1)/$$($(3)_ARCH)/ofiles
+$$($(3)_DEBUG_COUTPUTDIR)/%.o : %.c
+	mkdir -p $$($(3)_DEBUG_COUTPUTDIR)
+	clang -x c -arch $$($(3)_ARCH) -std=gnu11 -O0 -fasm-blocks -c $$< -o $$($(3)_DEBUG_COUTPUTDIR)/$$<.o
+
+$$($(3)_DEBUG_OUTPUTDIR)/$(MODULENAME): Makefile $(wildcard *.swift) $$($(3)_DEBUG_COUTPUTDIR)/*.o
```

Comment:
If you start with a clean directory, the *.o files won't be there, which means they won't be added to the dependencies, which means they won't be included in the build. I think something like this should work:

```csharp
# this should go outside the SwiftLibraryTemplate template
CFILES:=$(wildcard *.c)

# this should go inside the SwiftLibraryTemplate template
DEBUG_OBJFILES:=$$(foreach cfile,$(CFILES),$$($(3)_DEBUG_OUTPUTDIR)/$$(cfile:.c=.o))

$$($(3)_DEBUG_OUTPUTDIR)/$(MODULENAME): Makefile $(wildcard *.swift) $$(DEBUG_OBJFILES)
	...
```

Reply:
These sets of changes aren't quite right - I get `make: *** No rule to make target `bin/Debug/mac/x86_64/registeraccess.o', needed by `bin/Debug/mac/x86_64/XamGlue'.  Stop.`

Reply:
Something like this seems to work: https://gist.github.com/rolfbjarne/a52b046d93957e42d884651a73331e16